tests/data/virtual-server-route-externalname/externalname-svc.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: externalname-service
+spec:
+  type: ExternalName
+  externalName: nginx.com
```

Comment:
please note that if `nginx.com` is either (1) not available or (2) not resolved properly, the tests will see those cases as successful. Because in both cases, a request to `externalnginx.com` will return 502, which the tests expect.  Note that case (2) is hard to distinguish from the case of misconfigured resolver, which we configure using the  `resolver-addresses` configmap key.

It is possible to have a backend running inside the cluster and expose it as external service (assumung we can have coffee service in the default namespace):
```
kind: Service
apiVersion: v1
metadata:
  name: externalname-service
spec:
  type: ExternalName
  externalName: coffee.default.svc.cluster.local
```
```
kind: ConfigMap
apiVersion: v1
metadata:
  name: nginx-config
data:
  resolver-addresses: "kube-dns.kube-system.svc.cluster.local" 
```
This way we can always be sure that the backend is available and returns 200 repossess. As a result, we won't have (1) and (2).

I don't think we need to address (1) and (2) right now, but this is something we should address later.

Reply:
I'll address this in a separate PR, cause this requires a significant changes in pre-conditions fixtures.